site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -49,128 +58,138 @@ FusionAuth has several options for installation. Before installing FusionAuth, b
 
-{% include _image.liquid src="/assets/img/blogs/edit-application-screen.png" alt="Edit Application Screen" class="img-fluid full mt-2 mb-4" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/implementing-fusionauth-python/edit-application.png" alt="Edit Application Screen" class="
+img-fluid full mt-2 mb-4" figure=false %}
 
-If you click the edit icon on the "**FusionAuth**" application, you can see this application only has the option to toggle JWT on or off, or set Access and ID tokens to populate a lambda. For now, we will leave this as it is and hit **Save**.
+If you click the edit icon on the "FusionAuth" application, you can see this application only has the option to toggle JWT on or off, or set Access and Id tokens to populate a lambda. For now, we will leave this as it is and hit "**Save**"
+.
 
-{% include _image.liquid src="/assets/img/blogs/save-icon.png" alt="Save" class="img-fluid mb-4 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/save-icon.png" alt="Save" class="img-fluid mb-4 mx-auto d-block"
+figure=false %}
 
-We want to create a separate app for our Python application to utilize, so let's go ahead and do that now. Select the green plus sign at the top right of the page to add an App.
+We want to create a separate app for our Python application to utilize, so let's go ahead and do that now. Select the green plus sign at the top right of the page to "**Add**" an App.
 
-{% include _image.liquid src="/assets/img/blogs/add-button.png" alt="Add" class="img-fluid mt-2 mb-4 mx-auto d-block" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/add-button.png" alt="Add" class="img-fluid mt-2 mb-4 mx-auto d-block"
+figure=false %}
 
 You should come to a page that looks like this:
 
-{% include _image.liquid src="/assets/img/blogs/add-application-screen.png" alt="Add Application Screen" class="img-fluid full mb-4" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/implementing-fusionauth-python/add-application.png" alt="Add Application Screen" class="img-fluid
+full mb-4" figure=false %}
 
+Give your application a name. We will call ours *pythonapp*. The name is just for us to recognize our app in the FusionAuth UI and is for display purposes only. An Application Id will be specified automatically on "**Save**".
 
-Give your application a name. We will call ours "**pythonapp**". The name is just for us to recognize our app in the FusionAuth UI and is for display purposes only. An Application ID will be specified automatically on **Save**.
+At this point, you can explore the many application settings FusionAuth has to offer as there are lots more options than the FusionAuth default application. We will not need to change or add anything else to the app at this time. When you're ready, hit "**Save**".
 
+{% include _image.liquid src="/assets/img/blogs/save-icon.png" alt="Save" class="img-fluid mb-4 mt-2 mx-auto d-block"
+figure=false %}
```

Comment:
Remove all image classes except for `img-fluid`.

Reply:
fixed in latest commit